resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -546,3 +546,19 @@ cloud_debug.instrument.production_warning.checkbox_label=I confirm {0} is not a
 cloud_debug.instrument.production_warning.title=Warning!
 lambda.run_configuration.debug_host=Debug Host:
 lambda.sam.debugHost.tooltip=The host to connect the debugger to
+
+feedback.title=Send Quick Feedback...
```

Comment:
nit: use same phrasing everywhere. Either "send" or "submit"

Reply:
standardized on submit. also changed OK button to submit